dio/lib/src/options.dart

```diff
@@ -630,6 +633,11 @@ class RequestOptions extends _RequestConfig with OptionsMixin {
 
   /// {@macro dio.options.ProgressCallback}
   ProgressCallback? onSendProgress;
+
+  /// Get blob url
+  ///
+  /// Only work in web
+  bool blobUrl;
```

Comment:
Is that the only way to enable the Blob feature for the request on the Web? Could we use pattern matching to determine a Blob URL like `Uri.parse(input).scheme == 'blob'`?

Reply:
`BaseOptions` does [not support](https://github.com/cfug/dio/blob/4dec2237d14ecd9b27a5718e720c114ffacd81a9/dio/lib/src/options.dart#L143) Uri, it only takes string !

Reply:
It should be safe to parse to a URI though.

On a related note: This would probably a good API change for the next stable version. URLs should be URIs, not Strings

Reply:
I tested with `requestUri`, first it adds the address of the word `blob` and the download encounters an error.

Download address instead
`https://example.com/1.mp4`
It changes to this
`blob:https://example.com/1.mp4`